### DISCUSSION TITLE
How can i handle No Valid Authenticators Found without pop up message

### DISCUSSION THREAD

**Discussion by h1zqeel** [MasterKale/SimpleWebAuthn#218] [Troubleshooting]
if i register a user from another device and try to login in a new device, it shows the below error on `startAuthentication()`
how can i handle this without it showing this pop up
<img width="1440" alt="Screenshot 2022-06-22 at 12 28 15 AM" src="https://user-images.githubusercontent.com/26511203/174882111-d3061068-2439-4557-b32b-f0d3e48a5913.png">
@MasterKale

**Comment by h1zqeel** [comment]
NotAllowedError: No available authenticator recognized any of the allowed credentials
    at identifyAuthenticationError (index.js?56ff:163:20)
    at _callee$ (index.js?56ff:202:15)
    at tryCatch (runtime.js?ecd4:45:16)
    at Generator.invoke [as _invoke] (runtime.js?ecd4:274:1)
    at prototype.<computed> [as throw] (runtime.js?ecd4:97:1)
    at asyncGeneratorStep (index.js?56ff:2:1)
    at _throw (index.js?56ff:2:1)
the following error is thrown which is okay with me, but i dont want it to try to open the fingerPrint Menu if it happens

**Comment by MasterKale** [comment]
Short answer: You can't.

WebAuthn by design wants to preserve user privacy, and so there's no mechanism for an RP to silently query for if the machine recognizes a list of credential ID's to know ahead of time if auth has a chance to succeed. You need to prepare to invoke WebAuthn and handle the failure accordingly.

It's clunky and suboptimal from an RP's perspective, but then that's a common complaint by RP's looking to go passwordless 😂 😭

**Comment by MasterKale** [comment]
You'll probably be interested in this issue on the WebAuthn repo: https://github.com/w3c/webauthn/issues/1749
